news/2025/2025-04-08-beatmap-spotlights-revival-spring-2025.md

```diff
@@ -0,0 +1,269 @@
+- [siromaru - Imperial Prime Aggression (AHHHHHHHHHHHHHH) \[\[4K\] Extra\]](https://osu.ppy.sh/beatmapsets/2089498#mania/4571836), suggested by [Carpihat](https://osu.ppy.sh/users/10085090)
+- [X JAPAN - Kurenai (Alptraum) \[\[4K\] Crying in deep red\]](https://osu.ppy.sh/beatmapsets/2101873#mania/4409471), suggested by [Carpihat](https://osu.ppy.sh/users/10085090)
+- [Camellia - We Could Get More Machinegun Psystyle! (And More Genre Switches) (Rio1-) \[\[6K\] Relentless\]](https://osu.ppy.sh/beatmapsets/2211806#mania/4685601), suggested by [Antalf](https://osu.ppy.sh/users/8793773)
+- [Ludicin - Everlasting Eternity (Critical\_Star) \[\[7K\] Timeless\]](https://osu.ppy.sh/beatmapsets/2099753#mania/4404687), suggested by [Maxus](https://osu.ppy.sh/users/4335785)
+
+Playlist B, C and D will be available on the [Beatmap Spotlights article](/wiki/Beatmap_Spotlights) as we open the playlist lobbies, so stay tuned!
+
+## Beatmap Spotlight Curator applications
+
+None of this would be possible without our [curators](https://osu.ppy.sh/groups/48)' hard work and efforts.
+
+That being said, we are looking towards expanding the team further in order to make picking the beatmaps more deliberate and effective.
+
+Are you an experienced beatmapper? An outstanding player? An exceptional mappooler? A mix of each? Now is your time to shine! You can find the application form below, but be sure to fill it in before <!-- deadline --> if you'd like to join the ranks of the Beatmap Spotlights Curators, putting a brick in the project's years-long history!
+
+[**Click here to apply!**](https://forms.gle/MjGxXe8C3hY2vUHy9)
+
+## New communication and announcement system
+
+As of today, communications about Beatmap Spotlights will be moved to the [osu! Discord server](https://discord.com/invite/ppy) entirely.
+
+`#osu-spotlights` is your room to talk about everything related to Beatmap Spotlights.
+
+You can also now opt in for news about seasons, playlists and all sorts of general announcements regarding the project by assigning yourself a ping role in `#welcome` channel.
+
+## The future
+
+That surely was a lengthy news post. It definitely marks the start of a new era for the Beatmap Spotlights project.
+
+What do you want for the next season? What needs to be fixed? What are your opinions on the current system? Be sure to let us know in the comments or sparkle a talk in the dedicated Spotligths osu! Discord channel.
+
+We do hope you enjoy the brand-new implementations, and of course, we do hope that you stay with us for a long while.
+
+Now go ahead and set some scores, see you in the playlist lobbies!
+
+---
+
+Join us over at the [osu! Discord server](https://discord.com/invite/ppy) and assign yourself to a pingable role to keep up with changes and announcements. You can also find out more about the [Beatmap Spotlights](/wiki/Beatmap_Spotlights) on the osu! wiki.
```

Comment:
```suggestion
```
This literally already exists on line 253. Just add the wiki part to that.

Reply:
just reduced this to osu wiki for this very paragraph